docs/architecture/adr-016-validator-consensus-key-rotation.md

```diff
@@ -34,11 +34,11 @@ Also, it should be noted that this ADR includes only the simplest form of consen
     * In current Tendermint design, consensus key rotations are seen as power changes from LCD or IBC perspective
     * Therefore, to minimize unnecessary frequent key rotation behavior, we limited maximum number of rotation in recent unbonding period and also applied exponentially increasing rotation fee
 * limits
-    * a validator cannot rotate its consensus key more than `MaxConsPubKeyRotations` time for any unbonding period, to prevent spam.
+    * rotations are limited to 1 time in an unbonding window. In future rewrites of the staking module it could be made to happen more times than 1
     * parameters can be decided by governance and stored in genesis file.
 * key rotation fee
     * a validator should pay `KeyRotationFee` to rotate the consensus key which is calculated as below
-    * `KeyRotationFee` = (max(`VotingPowerPercentage` *100, 1)* `InitialKeyRotationFee`) * 2^(number of rotations in `ConsPubKeyRotationHistory` in recent unbonding period)
```

Comment:
nit: what "in recen unbonding period" means? In the above paragraph it says that only 1 rotation is allow per unbonding window.

Reply:
> In the above paragraph it says that only 1 rotation is allow per unbonding window.

Yeah, currently the rotations are allowed once in unbonding period, but this formula applies when we allow more than one rotation in unbonding period of time.